core/src/commands/moderation.cpp

```diff
@@ -630,3 +630,25 @@ void AOClient::cmdClearCM(int argc, QStringList argv)
     arup(ARUPType::CM, true);
     sendServerMessage("Removed all CMs from this area.");
 }
+
+void AOClient::cmdKickother(int argc, QStringList argv)
+{
+    Q_UNUSED(argc);
+    Q_UNUSED(argv);
+
+    QList<AOClient *> l_invoker_clients = server->getClientsByIpid(m_ipid);
+
+    int l_clients_kicked = 0;
+
+    for ( AOClient* l_client : l_invoker_clients ) {
```

Comment:
Variable name is not descriptive.
```suggestion
    for ( AOClient* l_client : l_multiclients) {
```